### PR TITLE
[author][new] adding domainr-search-box with npm auto update

### PR DESCRIPTION
**Pull request by iangilman**
Library from https://github.com/domainr/domainr-search-box for the https://domainr.com/ API.

### CODE REVIEW

File: ajax/libs/domainr-search-box/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "search",
+    "domain names",
+    "tld"
+  ],
+  "filename": "domainr-search-box.min.js",
+  "npmName": "domainr-search-box",
+  "npmFileMap": [{
+    "basePath": "/dist/",
+    "files": [
+      "*.*"
```
Comment (PeterDaveHello) [inline ajax/libs/domainr-search-box/package.json]: What about set to `**/*`?

**Comment by PeterDaveHello** [comment]
Would you please also provide minified css?

**Comment by iangilman** [comment]
@PeterDaveHello Thanks for the quick feedback! I've updated the patch with minified css (which is now build into our npm package) and the updated file matcher.

**Comment by PeterDaveHello** [comment]
Can you also squash them into only one commit? Thanks.

**Comment by iangilman** [comment]
I can't squash them using `git rebase` due to the sortable/Sortable issue(#3650), and I had trouble getting rebase to work using the sparseCheckout technique (https://github.com/cdnjs/cdnjs/blob/master/sparseCheckout.md), so I went ahead and made a new branch with a fresh copy and started a new pull request with that: https://github.com/cdnjs/cdnjs/pull/4739 I hope that works!

**Pull request closed without merging**